backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala

```diff
@@ -229,7 +229,8 @@ object BackendSettings extends BackendSettingsApi {
    */
   private def isCount1(plan: SparkPlan): Boolean = {
     plan match {
-      case exec: HashAggregateExec if exec.aggregateExpressions.forall(expression =>
+      case exec: HashAggregateExec if exec.aggregateExpressions.nonEmpty &&
+        exec.aggregateExpressions.forall(expression =>
```

Comment:
Can we use the blow code for this part? Assume we just expect one child for each aggregation expression and it should be Literal(1).  
```
  private def isCount1(plan: SparkPlan): Boolean = {
    plan match {
      case exec: HashAggregateExec if exec.aggregateExpressions.forall(expression =>
          expression match {
            case c: Count if c.children.size == 1 =>
              c.children.head.equals(Literal(1))
            case _ =>
              false
          }) =>
        true
      case _ =>
        false
    }
  }
```

Reply:
Thanks! Updated.